R/get_error_path.R

```diff
@@ -59,6 +87,11 @@ get_error_path <- function(schema, element = "target_metadata",
       unique()
   }
 
+  # If the path we are targeting is an an array (i.e. property name is followed
+  # by "items" and `append_item_n` is `TRUE`, (i.e. we want to include the
+  # index to a specific element in that object), then add an "\items" string to the
+  # end of the path where a location index will be added during instance
+  # path interpolation.
```

Comment:
🙏🏽 thank you!

Reply:
One suggestion I have would be to move out the subsequent validation checks to sentinel variables above the if statement. For example:

```r
  path_exists <- length(path) != 0
  is_array <- path_exists && any(grepl(paste(path, "items", sep = "/"), schema_paths))
  if (append_item_n && is_array) {
```

Reply:
Done